swagger-ui/index.html

```diff
@@ -39,7 +39,7 @@
     window.onload = function() {
       // Begin Swagger UI call region
       const ui = SwaggerUIBundle({
-        url: "./../swagger.json",
+        url: "https://petstore.swagger.io/v2/swagger.json",
```

Comment:
Not sure I understand the general flow of things, was linking to the general petstore json intentional here?

Reply:
URL is always overwritten by the query parameter passed.  This URL specified here will never matter.